react-native-sdk/update_dependencies.js

```diff
@@ -25,6 +25,15 @@ function updateDependencies() {
             updated = true;
         }
 
+        if (!semver.valid(packageJSON.dependencies[key])) {
+            packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
+            updated = true;
+
+            console.log(`
+⚠️We changed ${key} version number from ${packageJSON.dependencies[key]} to ${RNSDKpackageJSON.peerDependencies[key]}`
+            );
+        }
```

Comment:
You need to continue here. No point in doing more checks.

Reply:
But we need to update the version to a valid one, if that is the case, right?

Reply:
Updated.